src/Core/DefaultGeometryLayers.js

```diff
@@ -168,14 +209,28 @@ export function create3dTiles(id, options) {
 
     layer.url = options.url.href;
     layer.protocol = '3d-tiles';
-    layer.overrideMaterials = true;  // custom cesium shaders are not functional
+    layer.overrideMaterials = options.overrideMaterials === undefined ? true : options.overrideMaterials;
     layer.type = 'geometry';
     layer.visible = true;
-    layer.sseThreshold = 1;
+    layer.sseThreshold = options.sseThreshold || 16;
+    layer.cleanupDelay = options.cleanupDelay || 1000;
 
     return layer;
 }
 
+/**
+ * Creates a geometry layer to display a pointcloud.
+ * The pointcloud data must have been prepared using PotreeConverter.
+ *
+ * @function
+ * @param {string} id - identifer (must be unique amongst all layers id)
+ * @param {Object} options
+ * @param {string} options.url URL to the folder containing options.file
```

Comment:
"URL **of**"

Note: "URL to" may be correct, but seems awkward.

Note 2: we should choose between "url" and "URL". "URL" seems to be the correct form.